src/calculateRentalCost.js

```diff
@@ -4,7 +4,21 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const DAILY_RATE = 40;
+  const LONG_TERM_DISCOUNT = 50;
+  const SHORT_TERM_DISCOUNT = 20;
+  const LONG_TERM_RENTAL = 7;
+  const SHORT_TERM_RENTAL = 3;
```

Comment:
The use of constants for daily rate and discounts is excellent as it avoids hardcoding and makes the values easily configurable.